sandbox/tests/storage_test.rs

```diff
@@ -73,3 +73,91 @@ fn test_ext_storage_set(){
     assert_eq!(result, [0u8; 0]);
     assert_eq!(exp_value, Some(value.encode()));
 }
+
+#[test]
+fn test_ext_storage_read(){
```

Comment:
I think that it would be better if we have a couple of tests for the `read` so that we cover the different scenarios.
- First test can be testing that we can read the whole input (f.e having a value of 25 elements, buffer with length 25 and offset 0) and the buffer is populated correctly.
- The second one could be testing that the offset reading is working (f.e having a value of 25 elements, buffer with length 25 and offset 10) and the buffer is populated correctly with elements (buffer should have elements from 0 to 15 and from 15 to 25 should be zeros)
- The third one could be testing that if the buffer is smaller than the value it is populated. (f.e having a value of 25 elements, buffer with length 15 and offset 0 -> the buffer should have 15 elements populated)